test/integration/testdata/pilot.yaml.tmpl

```diff
@@ -50,8 +95,17 @@ spec:
         volumeMounts:
         - name: config-volume
           mountPath: /etc/istio/config
+{{if .DlvPort}}
+        - mountPath: "/data/delve"
+          name: dlv
+{{end}}
       volumes:
       - name: config-volume
         configMap:
           name: istio
+{{if .DlvPort}}
```

Comment:
There's quite a big template addition from what I can tell. I expect some documentation how to make use of these, too.